tox.ini

```diff
@@ -31,13 +30,18 @@ commands=
     native-blockchain-petersburg: pytest {posargs:tests/json-fixtures/test_blockchain.py --fork ConstantinopleFix}
     native-blockchain-metropolis: pytest {posargs:tests/json-fixtures/test_blockchain.py --fork Metropolis}
     native-blockchain-transition: pytest {posargs:tests/json-fixtures/test_blockchain.py -k BlockchainTests/TransitionTests}
+    lint: flake8 {toxinidir}/eth {toxinidir}/tests {toxinidir}/scripts
+    lint: mypy -p eth --config-file {toxinidir}/mypy.ini
 
-deps = .[eth-extra,test]
+deps =
+    .[eth-extra,test]
+    lint: .[eth,lint]
 
 basepython =
-    py35: python3.5
     py36: python3.6
     py37: python3.7
+setenv=
+    lint: MYPYPATH={toxinidir}:{toxinidir}/stubs
```

Comment:
just noting that we *should* be able to move away from this but need to double check what stubs we are using.

Reply:
Huh, I moved this over pretty blindly. There is no stubs folder... I'm just going to try removing it and see what happens.